frontend/utilities/helpers.ts

```diff
@@ -661,15 +661,21 @@ const DISK_ENCRYPTION_MESSAGES = {
   },
 };
 
-export const humanHostDiskEncryptionEnabled = (
-  platform?: string,
-  isDiskEncrypted = false
-): string => {
-  if (platform !== "windows" && platform !== "darwin") {
+export const getHostDiskEncryptionTooltipMessage = (
+  platform: "darwin" | "windows" | "chrome", // TODO: improve this type
```

Comment:
How about "linux"? Can we use or create an interface from `interfaces/platform.ts`

Reply:
We don't display disk encryption info for Linux, so it's not an option.

Having the typing like this is cleanest to me – if you feel strongly having an external type here is cleaner we can do that.